exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -45,7 +45,7 @@ drill.client: {
   supports-complex-types: true
 }
 
-// Directory is used as base for temporary storage of Dynamic UDF jars.
+// Directory is used as root for temporary storage of Dynamic UDF jars.
```

Comment:
Where does the user configure the suffix added to this directory to get the temp dir for a given cluster? For example, should we have something like:

drill.cluster-temp-dir: "${drill.tmp-dir}/${drill.exec.zk.root}"

Note that tmp itself is a shared resource. If it is in /tmp, then multiple clusters will use it. If it is in $DRILL_HOME/tmp, then still multiple clusters will use it. Only if it is configured as $DRILL_SITE/tmp can we be guaranteed only one cluster will use the directory (because only one cluster can use each $DRILL_SITE dir -- that's how $DRILL_SITE is defined.)